app/frontend/src/locales/es/translation.json

```diff
@@ -0,0 +1,21 @@
+{
+    "app": {
+        "title": "Habla con tus datos",
+        "footer": "Creado con Azure AI Search + Azure OpenAI",
+        "stopRecording": "Detener grabación",
+        "startRecording": "Comenzar grabación",
+        "stopConversation": "Detener conversación"
+    },
+    "status": {
+        "notRecordingMessage": "Pregunta cualquier cosa sobre los beneficios de los empleados de Contoso",
+        "conversationInProgress": "Conversación en progreso"
+    },
+    "history": {
+        "answerHistory": "Historial de respuestas",
+        "noHistory": "Aún no hay historial."
+    },
+    "groundingFiles": {
+        "title": "Archivos de fundamentación",
```

Comment:
This reads odd in Spanish to me, but I also don't know how to call it instead... Feels like an English term specific to AI, 'grounding files', not sure if there's a direct translation.

Reply:
"Sources"? (Fuentes??)

Reply:
I would say citations but they're not necessarily citations, right, since the LLM may or may not reference them all? Or maybe they are?